assets/sass/local.scss

```diff
@@ -291,7 +291,14 @@
   white-space: nowrap;
 }
 
-.govuk-input__wrapper {
-  width: 215px;
-  margin-bottom: 24px;
+.bapv-link-button, .bapv-link-button:active, .bapv-link-button:hover {
```

Comment:
Using buttons for the 'Unblock' action so they can submit forms, but styling them to look like links.

Reply:
Hmm, semantically I agree as it's submitting a form but confusing for the user?

Reply:
Yeah, I'd expect a link to go somewhere not submit a form. The other way around, styling a link as a button works fine because it's still an action. Might be worth chatting with design? I have no issue with it looking like a button as that's what it is but it's not my product 😀

Reply:
No worries - I'll raise the issue and see what people say... (tbh I've seen a fair number of DPS things do this so perhaps will be seen as an 'established pattern' for users 🤷‍♂️!)

Reply:
Yep, I seem to remember they had a designer who basically did what they wanted as I've seen it before and was asked if it was OK by another dev.